@here/harp-mapview/lib/MapViewTaskScheduler.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Task, TaskQueue } from "@here/harp-utils";
+
+import { MapView, TileTaskGroups } from "./MapView";
+
+export class MapViewTaskScheduler {
+    private m_taskQueue: TaskQueue;
+    private m_throttlingEnabled: boolean = false;
+
+    constructor(private m_mapView: MapView) {
```

Comment:
Please don't add whole `MapView` as dependency. As far as I can see it is only used for calling update. That could be solved with a callback similar to `DataSource.requestUpdate`

Reply:
its also used for the maxFps property, but Ill pass that to the Scheduler, so the dependency is not needed.